src/Api/AssetReferenceFileApi.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Akeneo\PimEnterprise\ApiClient\Api;
+
+use Akeneo\Pim\ApiClient\Client\ResourceClientInterface;
+use Akeneo\Pim\ApiClient\FileSystem\FileSystemInterface;
+use Akeneo\PimEnterprise\ApiClient\Exception\UploadAssetReferenceFileErrorException;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * API implementation to manage asset reference files.
+ *
+ * @author    Laurent Petard <laurent.petard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AssetReferenceFileApi implements AssetReferenceFileApiInterface
+{
+    const ASSET_REFERENCE_FILE_URI = '/api/rest/v1/assets/%s/reference-files/%s';
+    const ASSET_REFERENCE_FILE_DOWNLOAD_URI = '/api/rest/v1/assets/%s/reference-files/%s/download';
+
+    /** @var ResourceClientInterface */
+    private $resourceClient;
+
+    /** @var FileSystemInterface */
+    private $fileSystem;
+
+    /**
+     * @param ResourceClientInterface $resourceClient
+     * @param FileSystemInterface     $fileSystem
+     */
+    public function __construct(ResourceClientInterface $resourceClient, FileSystemInterface $fileSystem)
+    {
+        $this->resourceClient = $resourceClient;
+        $this->fileSystem = $fileSystem;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFromLocalizableAsset($assetCode, $localeCode)
```

Comment:
`getLocalizableAsset` / `getNotLocalizableAsset` ?

Reply:
Seen IRL: we get a reference file FROM an asset. Forget my comments about naming.